content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -51,9 +38,58 @@ After you deploy Kubernetes, obtain your KFP endpoint by following [these instru
 </details>
 
 <details>
-<summary><a name="kfp_qs_step2"></a><h2 style="display:inline;">Step 2: Create and run a simple pipeline using the KFP SDK</h2></summary>
+<summary><a name="kfp_qs_ui"></a><h2 style="display:inline;">Run a basic pipeline on the KFP Dashboard</h2></summary>
+
+Kubeflow Pipelines offers a few samples that you can use to try out
+Kubeflow Pipelines quickly. The steps below show you how to run a basic sample that
+includes some Python operations, but doesn't include a machine learning (ML) 
+workload:
+
+1. Click the name of the sample, **[Tutorial] Data passing in python components**, on the pipelines UI:
+  <img src="/docs/images/v2/click-pipeline-sample.png" 
+    alt="Pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+2. Click **Create experiment**:
```

Comment:
In this step, can we ask users to click `Create run` directly and Create experiment in the `NewRun` page? I am trying to navigate users to a common routine user journey, which means that users don't necessarily need to create a new experiment every time they want to create a run.